clippy_lints/src/utils/conf.rs

```diff
@@ -437,7 +437,7 @@ define_Conf! {
     ///
     /// The maximum size of the `Err`-variant in a `Result` returned from a function
     (large_error_threshold: u64 = 128),
-    /// Lint: MUTABLE_KEY_TYPE.
+    /// Lint: MUTABLE_KEY_TYPE, IFS_SAME_COND.
     ///
     /// A list of paths to types that should be treated like `Arc`, i.e. ignored but
```

Comment:
The docs imply to me that `Arc` and other pointer types like `Cell` are included in the list, even if they aren't part of the default configuration. Do you maybe want to add them?

Reply:
you mean adding Arc. Mutex, RWLock, Cell, RefCell into the list as well?

Reply:
Oh wait, I think did the opposite of what that config was designed to do... those types in the list were supposed to be checked.

I got confused by the name lol

Reply:
Oh wait, I think I did the opposite of what that config was designed to do... those types in the list were supposed to be checked.

I got confused by the name lol

Reply:
Now that you say it, it's a bit confusing :sweat_smile:. I guess we'll leave it like this now, since it's not new.